### PR TITLE
clean up calendar and remove legacy meetings

### PR DESCRIPTION
**Pull request by thesuperzapper**
This PR resolves https://github.com/kubeflow/community/issues/630

It also makes it more clear which meetings are active and warns users not to remove recurring meetings from the file (as this leaves them in the calendar forever).

### CODE REVIEW

**Comment by thesuperzapper** [comment]
@jbottum @juliusvonkohout @zijianjoy this PR removes meetings from the calendar which are no longer happening, I would appreciate your approval.

**Comment by thesuperzapper** [comment]
@akgraner just making you aware of this PR, so we can merge it before your one, to prevent removed events being re-added.

**Comment by juliusvonkohout** [comment]
i did not check every detail but /lgtm

**Review by chensun — COMMENTED** [review]
The diff makes it hard to review what's being removed. Can you avoid unnecessary changes/reordering?

**Comment by thesuperzapper** [comment]
@chensun the reordering (and docs update) is the main point of the PR, as it makes it easier to see which are the "current meetings"  and also warns users about how the file actually works at the top.

The only meeting "change" is setting an end date for the `Kubeflow Feature Store SIG Meeting (US/Asia friendly)` meeting (which has not been happening for many years, as seen in https://github.com/kubeflow/community/issues/630).

PS: I have rebased it after you merged https://github.com/kubeflow/community/pull/641, but would appreciate merging this now so we don't have to rebase again.

**Comment by thesuperzapper** [comment]
I have also removed the `until` date on `Kubeflow Notebooks WG Meeting` and `Kubeflow Release Team Meeting (CET, US friendly)` and `Kubeflow Security Team Call (US West/APAC)`, which don't need an end date (which is probably why the diff is confusing).

**Review by chensun — APPROVED** [review]
/lgtm
/approve

**Pull request merged**
